backend/controllers/email.controller.js

```diff
@@ -53,9 +53,9 @@ const createProdSMTPTransport = async () => {
   return smtpTransport;
 };
 
-async function sendMail(smtpTransport, email, token) {
+async function sendMail(smtpTransport, email, token, origin) {
   const encodedToken = encodeURIComponent(token);
-  const emailLink = `https://tinyurl.com/nyqxd/handleauth?token=${encodedToken}&signIn=true`;
+  const emailLink = `${origin}/handleauth?token=${encodedToken}&signIn=true`;
```

Comment:
Did you test this using an email provider? The reason this was changed was that this broke for localhost. I was unfortunately glued to doing some sort of redirection.

Reply:
I am going to ask for you to at least verify that it works end2end for local development before approving.

Reply:
@nickbeaird Yes. This worked on my localhost magic link that was generated for me was. Was sent to yahoo email and received. Linked worked for me on login.

http://localhost:3000/handleauth?token=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IjVmN2NmYzQ3ZjQ3ZGY2YzFmZmYwNTZlZiIsInJvbGUiOiJhZG1pbiIsImlhdCI6MTYwMzIzNjMxNiwiZXhwIjoxNjAzMjM3MjE2fQ.TzqekHoKPtHzkhDa2r8s8Y5OkGlPUVOfTQ-TlkUwvE8&signIn=true